src/components/Browse/brandOption.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { toggleBrowseBrand } from '../../redux/actions'
+import PropTypes from 'prop-types'
+
+export default function BrandOption ({ brand }) {
+  const dispatch = useDispatch()
+
+  const brands = useSelector(state => state.browse.filters.brands)
```

Comment:
I think we should revisit how many global state variables we have at some point, its probably better to limit some of these in scope (that would be for another PR)

Reply:
I think I agree when it comes to singleton states (like size and currency), but here the `brands` state is part of the larger filters state which itself is part of the browse state so it's valid imo